twisted/internet/base.py

```diff
@@ -466,6 +466,7 @@ class ReactorBase(object):
     _registerAsIOThread = True
 
     _stopped = True
+    _lastSignal = None
```

Comment:
This variable is either an int (on posix systems) or a bool (Windows).

How do I document that?  Like this?

`C{int}|C{bool}`